tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,36 @@
+import os
+from queue import Queue
+from shutil import which
+from unittest.case import skipIf
+
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
+
+
+@skipIf(which('phpstan') is None, 'PHPStan is not installed')
+class PHPStanBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('test section')
+        self.uut = PHPStanBear(self.section, Queue())
+        self.test_file1 = os.path.join(os.path.dirname(__file__),
+                                       'test_files',
+                                       'phplint_test1.php')
+        self.test_file2 = os.path.join(os.path.dirname(__file__),
+                                       'test_files',
+                                       'phplint_test2.php')
+
+    def test_run(self):
+        # Test a file with errors and warnings
+        self.check_validity(
```

Comment:
@satwikkansal Well I have used `check_validity` as one of the other bear test file PHPLintBearTest uses those two test files (i.e.`test_files/phplint_test1.php` & `test_files/phplint_test1.php`).So if we have the resources why not use them.Should i really change it to `check_results` ? Cause i have read in the docs we can use any of those 3 methods to test a bear.